src/modules/apps/link.ts

```diff
@@ -342,8 +342,12 @@ export default async (options) => {
   try {
     const buildTrigger = performInitialLink.bind(this, appId, builder, extraData)
     const [subject] = appId.split('@')
-    const { unlisten } = await listenBuild(subject, buildTrigger, { waitCompletion: false, onBuild, onError })
-    unlistenBuild = unlisten
+    if (options.watch === false) {
```

Comment:
`if (!options.watch)` doesn't work? BTW, can't you deconstruct this option in the beggining of the function?

Reply:
☝️ `if (!options.watch)` will evaluate to `true` if `watch` is nil. I think a strict comparison with `false` might be good in this case?